Tests/BotTests/MergeServiceTests.swift

```diff
@@ -673,6 +674,111 @@ class MergeServiceTests: XCTestCase {
         )
     }
 
+    func test_changing_pull_request_priorities() {
```

Comment:
Thanks for adding these tests. Gives much better confidence.

Reply:
I actually found a small bug in my algorithm while I was adding them, so it reassured me too 😉